can_bus/quadruna/CRIT/CRIT_alerts.json

```diff
@@ -54,5 +48,11 @@
             "id":254,
             "description" : "Example"
         }
-}
+    },
+    "faults": {
+        "MissingBMSHeartbeat": {
+            "id" :251,
+            "description" : "Example"
+        }
+    }
```

Comment:
What's the justification for leaving the BMS missing heartbeat a fault and moving the rest to warnings? Not against it, just wanna make sure there is a good reason